### ISSUE TITLE
Upgrade pip to 9.0.1

### ISSUE THREAD

**Issue by slozier** [IronLanguages/main#1650]
We should upgrade the pip we distribute to 9.0.1. Whenever people currently use pip they see the following message:
> You are using pip version 8.1.1, however version 9.0.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' command.

and then try to upgrade which tends to cause more issues to pop up.

**Comment by slide** [comment]
Yes, but trying 9.0.1 causes more issues...I would almost rather patch the version of pip we have to not print that message.

**Comment by slozier** [comment]
Hmm, I managed to install and use 9.0.1 over here. I'm not familiar with what tweaks had to be done to pip so maybe I'm missing something. Will do more testing tomorrow.

**Comment by slozier** [comment]
Alright, I ran a few more tests and am hitting issues as well. Let's go with patching the version of pip to not display.
